joern-cli/frontends/csharpsrc2cpg/src/main/scala/io/joern/csharpsrc2cpg/astcreation/AstForStatementsCreator.scala

```diff
@@ -210,4 +212,14 @@ trait AstForStatementsCreator(implicit withSchemaValidation: ValidationMode) { t
 
     identifierAst :+ gotoAst
   }
+
+  private def astForReturnStatement(returnStmt: DotNetNodeInfo): Seq[Ast] = {
+    val identifierAst = Option(returnStmt.json(ParserKeys.Expression)) match
```

Comment:
We're trying to move away from significant indentation as I've picked up from some reviews this week, so we should bring in braces for match statements we create, i.e., 
```
match {
...
}
```